TbsZip/tbszip.php

```diff
@@ -17,6 +17,26 @@
 
 class clsTbsZip {
 
+  private $Meth8Ok;
```

Comment:
FWIW, I googled around a little bit. "TBS" stands for "Tiny But Small" (https://www.tinybutstrong.com/) - they seem to have a few inter-related projects (eg https://github.com/Skrol29/tbszip and https://github.com/Skrol29/opentbs and https://github.com/Skrol29/tinybutstrong). The `tbszip.git` hasn't been updated in a long time, but I suspect it's still used - because the other repos have been updated.

Note, eg, OpenTBS was updated 2 months ago - and in that project, the main class [extends clsTbsZip and uses `$this->Meth8Ok`](https://github.com/Skrol29/opentbs/blob/89aea4db66bc76808f70e7fc7bd37980da0f5011/tbs_plugin_opentbs.php#L138C1-L138C89). So my guess is that - to get this upstreamed - it probably needs to be `protected` or `public`.

(cc @Skrol29)